e2e/support/helpers/e2e-viz-settings-helpers.js

```diff
@@ -13,3 +13,10 @@ export function openSeriesSettings(field, isBreakout = false) {
       .click();
   }
 }
+
+export function openVizType(tabName = "Chart") {
+  cy.findByTestId("viz-settings-button").click();
+  if (tabName) {
```

Comment:
Design feedback: it feels weird that clicking 1 of these 3 buttons makes this control disappear.
With [`SegmentedControl`](https://mantine.dev/core/segmented-control/) one expects to still see this control after interacting with it, but here when I click the "Editor" button the control disappears.

https://github.com/user-attachments/assets/2319273a-1205-4e84-aaec-1f3b2f2b6354

Reply:
To pile on to this: I don't love that the segmented control animates when it appears after clicking Visualize.


https://github.com/user-attachments/assets/5c3fc344-04eb-4fc1-8f29-cf3bc8c1cd9a

Reply:
Yes, this shouldn't be the case. The editor segment should get selected. The segmented control should only morph back into a button **after** the query is edited.

Reply:
clear, thanks!